tests/5.1/target/test_target_memcpy_rect_async_depobj.F90

```diff
@@ -75,10 +76,10 @@ INTEGER FUNCTION test_memcpy_rect_async_depobj()
 
     !$omp taskwait depend(depobj: obj)
     !$omp target is_device_ptr(devRect) device(t) depend(depobj: obj)
+    CALL c_f_pointer(devRect, fptr, [M*N])
     DO i=1, N
       DO j=1, M
-        CALL c_f_pointer(devRect, fptr, [M*N])
-        fptr(i*M+j) = fptr(i*M+j) * 2 ! initialize data
+        fptr((i-1)*M+j) = fptr((i-1)*M+j) * 2 ! initialize data
       END DO
     END DO
     !$omp end target
```

Comment:
```suggestion
    fptr => null()
    !$omp end target
```
Reason — I forgot to set the pointer back to disassociated before exiting the target region.

OpenMP requires in restrictions to the **target** construct:

_"If the association status of a list item with the POINTER attribute that appears in a **map** clause on the construct is disassociated upon entry to the target region, the list item must be disassociated upon exit from the region."_